go/vt/topo/k8stopo/version.go

```diff
@@ -36,5 +38,5 @@ func VersionFromInt(version int64) topo.Version {
 	if version == -1 {
 		return nil
 	}
-	return KubernetesVersion(version)
+	return KubernetesVersion(fmt.Sprint(version))
```

Comment:
I think it's better to use `strconv.FormatInt(version, 10)` because `fmt.Sprint()` first has to parse the argument and hence it's slightly slower than `strconv.FormatInt`. I did a small benchmark where you can see it:

Test: 

```go
func BenchmarkVersion(b *testing.B) {
	b.ResetTimer()

	// run the Fib function b.N times
	for n := 0; n < b.N; n++ {
		_ = VersionFromInt(100000)
	}
}
```

And then run it via:

```
$ go test -run=NONE -bench=. ./... > old.txt
$ go test -run=NONE -bench=. ./... > new.txt
$ benchstat old.txt new.txt
name        old time/op    new time/op    delta
Version-16    82.8ns ± 0%    24.6ns ± 0%   ~     (p=1.000 n=1+1)
```


Using `strconv.FormatInt(version, 10)` is slightly faster. It's not big in the grand schema, but the `fmt` package is also used mostly for formatting, so I think it's better to use `strconv` here.

Reply:
Oh wow thank you!! There's some other tests I fixed in other branches with the `fmt.Sprint` solution, which I'll fix up in another branch. In fact I'm actually going to just split this change out as well.